core/src/DialogBox.html

```diff
@@ -0,0 +1,203 @@
+                    server: { type: 'string', label: 'Server', isEditable: false, value: '' }
+                }
+            },
+            language: {
+                label: 'Language & Region',
+                sublabel: 'EN | Time Format: 12h',
+                icon: false,
+                title: 'Language & Region',
+                settings: {
+                    language: {
+                        type: 'enum',
+                        label: 'Language and Region',
+                        options: ['German', 'English', 'Spanish', 'French'],
+                        description: 'After you save your settings, the browser will refresh for the new language to take effect.'
+                    },
+                    date: { type: 'string', label: 'Date Format' },
+                    time: { type: 'enum', label: 'Time Format', options: ['12 h', '24 h'] }
+                }
+            },
+            privacy: {
+                label: 'Privacy',
+                title: 'Privacy',
+                settings: {
+                    policy: { type: 'string', label: 'Privacy policy has not been defined.' }
+                }
+            }
+        }
+    }
+
+    const mockSavedSettings = {
+        userSettingGroups: {
+            userAccount: {
+                label: 'User Account',
+                sublabel: 'username',
+                icon: 'icon.jpg',
+                title: 'title',
+                settings: {
+                    name: { type: 'string', label: 'Name', isEditable: true, value: 'Luigi' },
+                    email: { type: 'string', label: 'E-Mail', isEditable: false, value: 'luigi@luigi.de' },
+                    server: { type: 'string', label: 'Server', isEditable: false, value: 'luigi.server' }
+                }
+            },
+            language: {
+                label: 'Language & Region',
+                sublabel: 'EN | Time Format: 12h',
+                icon: false,
+                title: 'Language & Region',
+                settings: {
+                    language: {
+                        type: 'enum',
+                        label: 'Language and Region',
+                        options: ['German', 'English', 'Spanish', 'French', 'Italian'],
+                        description: 'After you save your settings, the browser will refresh for the new language to take effect.',
+                        value: 'Italian'
+                    },
+                    date: { type: 'string', label: 'Date Format' },
+                    time: { type: 'enum', label: 'Time Format', options: ['12 h', '24 h'], value='12 h' }
+                }
+            },
+            privacy: {
+                label: 'Privacy',
+                title: 'Privacy',
+                settings: {
+                    policy: { type: 'string', label: 'Privacy policy has not been defined.' }
+                }
+            }
+        }
+    }
```

Comment:
Quick suggestion, would it be pe possible to create a directory in src/schemas where we can add a new file userSetting.js and export userSettingsSchema and mockSavedSettings?  (or create two different files (one for the schmea and the other for the mock)...So here we could just import the two objects.

It's be easier in the future to localize/change the schema and it would reduce size of DialalogBox.xml

Reply:
It's a left over. I'll deleted this file. Schema has to be defined in luigi config settings section and how a core developer structure the settings is up to the developer.